src/main/kotlin/io/spine/code/proto/CodeGeneratorRequestWriter.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2024, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Let's use the fresh copyright header, with `https` in this line.

It's here and in all newly created files.